bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/api/MessageReceivedEvent.java

```diff
@@ -16,7 +16,8 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 /**
- * The {@link DataPointChangedEvent} is an event container for data point changes
+ * The {@link org.openhab.binding.ism8.server.DataPointChangedEvent}
+ * is an event container for data point changes
```

Comment:
Please undo this change, you added a comment from the wrong binding (ism8)

Reply:
Ok, the proposed change is wrong. But it seems it has already been wrong before the change.
Shouldn't it be ```MessageReceivedEvent````?